ui/components/ui/chip/chip.scss

```diff
@@ -42,6 +42,8 @@
     border: none;
     background: transparent;
     text-align: center;
+    width: 100%;
```

Comment:
Does this have any unintended consequences elsewhere chips are used? If no, what did this solve?

Reply:
Currently this is the first and only use of the ChipsWithInput. And without the width set this way: 
<img width="218" alt="Screen Shot 2021-07-01 at 4 04 28 PM" src="https://user-images.githubusercontent.com/34557516/124189379-3c797e00-da86-11eb-97cc-5d7abdda829a.png">

Reply:
haha, didn't expand the styles to see it was specific to the input. Okay, thanks for humoring me. 👍🏻